### DISCUSSION TITLE
ANNO SCOLASTICO 2024/2025

### DISCUSSION THREAD

**Discussion by colellagabriele889** [iisgiua/giuaschool#743] [Richieste di aiuto]
Quando si potrà effettuare il cambio del nuovo anno 2024/2025? Siamo a fine mese ma nessun aggiornamento è disponibile e il  nostro Istituto ha necessità di effettuare il cambio d'anno.
E cosa prevede la nuova versione 1.5.9?

Attendiamo Vostre
Cordiali Saluti

**Comment by trinko** [comment]
Mi spiace, si fa quel che si può.
Comunque il cambio d'anno va effettuato a ridosso dell'inizio del nuovo anno scolastico, quindi poco prima del primo settembre.

**Reply by colellagabriele889** [reply]
Va bene, la ringrazio per la sua risposta. Cosa prevedere la versione 1.5.9 e quando uscirà?
